### DISCUSSION TITLE
How to add minutes on self-hosted?

### DISCUSSION THREAD

**Discussion by DaveWhitmer** [FIRST-Tech-Challenge/fmltc#243] [Q&A] [answered]
How do I add minutes to our team?  We are self hosting, but it shows that we have a minute limit that counts down as we train models. 
 Thanks.

**Comment by uvidyadharan** [comment, marked as answer]
If you go to the `<BASE-URL>/admin` page you can increment  or reset team training times. The page should look something like this.
![image](https://user-images.githubusercontent.com/55924756/147304756-3f89ac2c-15a6-49dc-814e-e6105545d478.png)

Example; https://fmltc.uc.r.appspot.com/admin

**Reply by DaveWhitmer** [reply]
Thanks so much for the quick reply!

**Reply by uvidyadharan** [reply]
No problem.

**Comment by lizlooney** [comment]
You can also go to https://console.cloud.google.com/firestore/data?project=<your project> and:

1. In the left column (under Root), click on Team.
2. The middle column will show a "document" for each team that uses the system. Click on the team you want to add minutes to.
3. The right column will show the database entity for that team. Click on remaining_training_minutes.
4. In the dialog that pops up, change the field value to whatever you want. It doesn't have to be a small number; you can set it to 1000000 if you want.
5. Click update.
